sdk/eventgrid/azure-eventgrid/samples/async_samples/sample_publish_to_channel_async.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+FILE: sample_publish_to_channel_async.py
+DESCRIPTION:
+    These samples demonstrate creating a list of CloudEvents and sending them as a list to a partner channel.
+USAGE:
+    python sample_publish_to_channel_async.py
+    Set the environment variables with your own values before running the sample:
+    1) EVENTGRID_PARTNER_NAMESPACE_TOPIC_KEY - The access key of your eventgrid account.
+    2) EVENTGRID_PARTNER_NAMESPACE_TOPIC_ENDPOINT - The topic hostname. Typically it exists in the format
+    "https://<YOUR-TOPIC-NAME>.<REGION-NAME>.eventgrid.azure.net/api/events".
+    3) EVENTGRID_PARTNER_CHANNEL_NAME - The name of the channel to which the event should be published.
+"""
+# [START publish_cloud_event_to_topic]
+import os
+import asyncio
+from azure.eventgrid.aio import EventGridPublisherClient
+from azure.core.credentials import AzureKeyCredential
```

Comment:
wondering if this needs to be an aio credential?

Reply:
nope - AzureKeyCredential doesn't have to be aio